### PR TITLE
Dev (implemented all project features)

### PR DESCRIPTION
**Pull request by yosefHesham**
**Brief of what has been done**
Created a responsive website that contains 2 pages (Home, and about), with the ability to navigate between them.
--
**Home Page**
- Created a headline section, which contains the webpage logo and description,.etc.
- Create the main program section, which contains details about the summit content.
- Created speakers section and render them with JS.
---
**About Page**
- Created a headline section, which contains the webpage logo and description,.etc.
- Created a section that describes how the logo was created.
- Created a section for past-summits

Video link : [video demo](https://www.loom.com/share/f1f5f7a1f93d4f90b93c8c894c1621c2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,58 @@
...
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
 - Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Hat tip to [Kyrillos](https://github.com/Bondok6) who provided me with the assets.
+- The design is following [Creative Commons](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```
Comment (mateo951) [inline README.md:55]: Great! You are giving appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) ✔

File: README.md [hunk 2]
```diff
@@ -1,72 +1,58 @@
...
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Live Demo Link](https://yosefhesham.github.io/capstone1/)
```
Comment (mateo951) [inline README.md:20]: Your live link is functional! Well done using GitHub Pages to deploy your website 💣

File: lib/scripts/top-bar.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  <img
+    class="tool-bar-item"
+    src="images/social/twitter.svg"
+    alt="twitter-icon"
+  />
+  <p class="tool-bar-item">English</p>
+  <p class="tool-bar-item">My Page</p>
+  <p class="tool-bar-item last-item">Logout</p>
+</div>
+<div class="navbar">
```
Comment (mateo951) [inline lib/scripts/top-bar.js]: Both pages contain the same navbar and footer 🚀

Comment (mateo951) [reply lib/scripts/top-bar.js]: It's important that your HTML code follows proper semantics 🧐 Please change this `div` tag into a `nav` tag to improve this aspect of your project 🙌

File: about.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        <div class="past-summit">
+          <img src="images/acpc.jpg" alt="" />
+          <div class="overlay">
+            <p>2013</p>
+            <p>ACM ICPC Summit in Egypt</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
```
Comment (mateo951) [inline about.html:81]: Nice 💪😎 About has a dark background and Home a white one ✔

File: lib/styles/index.css [hunk 5]
```diff
@@ -0,0 +1,485 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Kumbh+Sans:wght@400;700&family=Lato:ital,wght@0,400;0,700;1,400&display=swap");
+
+@font-face {
+  font-family: "cocogoose";
+  src: url("fonts/cocogoose/");
+}
+
+:root {
+  --grey: #d3d3d3;
+  --black: #272a31;
```
Comment (mateo951) [inline lib/styles/index.css:10]: The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 🔥 Well done!

File: lib/styles/index.css [hunk 6]
```diff
@@ -0,0 +1,485 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Kumbh+Sans:wght@400;700&family=Lato:ital,wght@0,400;0,700;1,400&display=swap");
+
+@font-face {
+  font-family: "cocogoose";
```
Comment (mateo951) [inline lib/styles/index.css:4]: The pages uses the same fonts as the original design ✔

File: index.html [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+          <p class="content-description">
+            Show us your mind art, challenge yourself and many other students
+            soving programming contest
+          </p>
+        </div>
+      </div>
+      <a class="join-us" href=""> Join ACM ICPC Summit 2022</a>
+      <a href="" class="see-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="speakers">
```
Comment (mateo951) [inline index.html:73]: Nice semantics 🙌

File: lib/scripts/speakers.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import speakers from './speakers_data.js';
+
+const speakersWrapper = document.querySelector('.speakers-wrapper');
+const showMore = document.querySelector('.more-button');
+function addSpeakers() {
+  for (let i = 0; i < speakers.length; i += 1) {
+    const temp = document.createElement('temp');
+    const speaker = speakers[i];
+    temp.innerHTML = `<article class="speaker ${i > 1 ? 'expanded' : ''}">
```
Comment (mateo951) [inline lib/scripts/speakers.js:9]: Well done 🤯 Again, great use of propper tags to have good semantics 🤓

File: index.html [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+        </ul>
+        <div class="close">Close</div>
+      </div>
+      <div class="desktop-bar">
+
+      </div>
+      <div class="content-wrapper">
+        <p class="hint">"Think twice, code once"</p>
+        <a href="#" class="logo">
+          <h1>ACM ICPC Summit 2022</h1>
```
Comment (mateo951) [inline index.html:31]: There's at least 1, but no more than 1, `h1` tag in each page 😁

File: lib/scripts/speakers_data.js [hunk 10]
```diff
@@ -0,0 +1,43 @@
+const speakers = [
```
Comment (mateo951) [inline lib/scripts/speakers_data.js:1]: The section "Featured speakers" is generated dynamically with JavaScript 🔥

File: lib/scripts/speakers.js [hunk 11]
```diff
@@ -0,0 +1,45 @@
+import speakers from './speakers_data.js';
```
Comment (mateo951) [inline lib/scripts/speakers.js:1]: Love your code/file organization 💡 Splendid work ✔

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @yosefHesham, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements 👌
- Great Video Presentation 🧐
- JS implementation for dynamic object creation ✔
- Design is really good 🚀

I really enjoyed reviewing your capstone project 👀 You have done a fantastic job 🤓
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: lib/scripts/speakers.js (hunk 11 above)
Comment (yosefHesham) [reply lib/scripts/speakers.js:1]: thanks :heart:

**Review by Kingstalux — APPROVED** [review]
Hi @yosefHesham, 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6d43496a436773336e575157664a5a7650412f67697068792d646f776e73697a65642d6c617267652e676966

## Highlights ⚡ 
✅ Good video presentation 👌
✅ Great UI design 🔥 
✅ Professional README file 👍 
✅ Good commit messages and pull request description 💯 

Good job implementing the required change 👍 Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :ship: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**